contracts/evm/contracts/xcall/CallService.sol

```diff
@@ -367,24 +455,13 @@ contract CallService is IBSH, ICallService, IFeeManage, Initializable {
         } else {
             require(msg.sender == defaultConnections[fromNID], "NotAuthorized");
         }
-
         uint256 reqId = getNextReqId();
-        proxyReqs[reqId] = Types.ProxyRequest(
-            req.from,
-            req.to,
-            req.sn,
-            req.rollback,
-            dataHash,
-            req.protocols
-        );
-
+        proxyReqs[reqId] = req;
```

Comment:
Req with its data hashed should be stored here, is there no testcase that catches this?

Reply:
Actually I hashed and compare data in executeCall, in solidity this would require new structure just to store hashed data, is it necessary to hash it here?

Reply:
But then you hash both sides at that point. This change was made so that we store alot less data, which saves alot of gas. So it is quite important. 

If we stored the data the data would not be needed at all in executeCall

Reply:
Oh okay, will revert the changes